### PR TITLE
[CI] add missing comment in e2e test retry

### PR DESCRIPTION
**Pull request by sausagee**
## Motivation
Add the missing comment in the e2e test retry logic.  Also add 10 sec
between retries.

## Test Plan
Test in CI in-place.

### CODE REVIEW

**Comment by rexhoffman** [comment]
I think you should turn off the set -x output now that these have landed.   Visually hard to parse the failure message at the end of a test run:

https://app.circleci.com/pipelines/github/libra/libra/14462/workflows/2c200a16-da07-4981-b48f-14fd95d77ed0/jobs/64504/parallel-runs/5?filterBy=ALL

File: .circleci/config.yml [hunk 1]
```diff
@@ -286,6 +289,7 @@ jobs:
                   $libratest $target --test-threads 1 --exact --nocapture
                 status=$?
                 retry=$((retry + 1))
+                sleep 10
```
Comment (rexhoffman) [inline .circleci/config.yml:292]: not sure why the sleep 10?

Comment (rexhoffman) [reply .circleci/config.yml:292]: seems safe -- probably to give background processes time to die?

**Comment by rexhoffman** [comment]
doh, ment to delegate.... shrug.

File: .circleci/config.yml (hunk 1 above)
Comment (sausagee) [reply .circleci/config.yml:292]: Yep. It's insurance.

**Comment by sausagee** [comment]
> I think you should turn off the set -x output now that these have landed. Visually hard to parse the failure message at the end of a test run:
> 
> https://app.circleci.com/pipelines/github/libra/libra/14462/workflows/2c200a16-da07-4981-b48f-14fd95d77ed0/jobs/64504/parallel-runs/5?filterBy=ALL

SG
